examples/dbinit.py

```diff
@@ -0,0 +1,30 @@
+"""
+Register nodes with HaaS.
+
+This is intended to be used as a template for either creating a mock HaaS setup
+for development or to be modified to register real-life nodes that follow a
+particular pattern.
+"""
```

Comment:
Perhaps add a little description of the environment this file is geared towards in particular---using the complex_vlan driver with a switch named 'R10SW1' and ports that look like 'GI1/0/15'.  In fact, listing the relevant options in the config file might be good.